content/How to customize fonts in React Native - Manuela Viola.md

```diff
@@ -0,0 +1,109 @@
+# How to customize fonts in React Native
+
+![Main image](images/customize_fonts_main.png)
+
+## Introduction
+
+Like every newbie in Rootstrap, my first assignment was to develop Target MVD application, in my case, in React Native. While doing this, one of the first things I tried was to set the family font that the application uses, and I found out it wasn't as easy as I expected.
+
+The biggest issue was that all the tutorials said to excecute "react-native link" command and after executing it, the application would stop building successfully.
+
+## Overview
+
+In this tutorial you will learn how to add and link fonts manually, without using the link command.
+
+First you will see all the configurations that need to be done in order to be able to use customize fonts and then you will learn its usage in a simple application.
+
+## Configuration
+
+### Create Project
+
+First of all, you should create a project in react native. To do that, initiate terminal in the folder where you want to create your project and excecute:
```

Comment:
To do that, open the terminal and navigate to the folder you want to create your project in, and execute: